api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java

```diff
@@ -252,20 +251,22 @@ private static String buildDemoSql(
     switch (CriteriaType.valueOf(param.getType())) {
       case AGE:
         Attribute attribute = param.getAttributes().get(0);
-        String ageNamedParameter1 =
+        String ageNamedParameter =
             QueryParameterUtil.addQueryParameterValue(
                 queryParams, QueryParameterValue.int64(new Long(attribute.getOperands().get(0))));
         if (attribute.getOperands().size() > 1) {
           String ageNamedParameter2 =
               QueryParameterUtil.addQueryParameterValue(
                   queryParams, QueryParameterValue.int64(new Long(attribute.getOperands().get(1))));
-          ageNamedParameter1 = ageNamedParameter1 + AND + ageNamedParameter2;
+          ageNamedParameter = ageNamedParameter + AND + ageNamedParameter2;
```

Comment:
Now that ageNamedParameter1 is rename to ageNamedParameter, should ageNamedParameter2 be updated as well

Reply:
done.